Samples/GettingStarted/BinaryClassification_SentimentAnalysis/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BinaryClassification_SentimentAnalysis
+{
+    internal static class Program
+    {
+        private static string AppPath => Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]);
+        private static string TrainDataPath => Path.Combine(AppPath, @"..\..\..\..\datasets\", "imdb_labelled.txt");
+        private static string TestDataPath => Path.Combine(AppPath, @"..\..\..\..\datasets\", "yelp_labelled.txt");
+        private static string ModelPath => Path.Combine(AppPath, "Models", "SentimentModel.zip");
+
+        private static async Task Main(string[] args)
+        {
+            var model = await TrainAsync();
+
+            Evaluate(model);
+
+            var predictions = model.Predict(TestSentimentData.Sentiments);
+
+            var sentimentsAndPredictions =
+                TestSentimentData.Sentiments.Zip(predictions, (sentiment, prediction) => (sentiment, prediction));
+            foreach (var item in sentimentsAndPredictions)
+            {
+                Console.WriteLine(
+                    $"Sentiment: {item.sentiment.SentimentText} | Prediction: {(item.prediction.Sentiment ? "Positive" : "Negative")} sentiment");
+            }
+
+            Console.ReadLine();
+        }
+
+        public static async Task<PredictionModel<SentimentData, SentimentPrediction>> TrainAsync()
+        {
+            // LearningPipeline allows us to add steps in order to keep everything together 
+            // during the learning process.  
+            var pipeline = new LearningPipeline();
+
+            // The TextLoader loads a dataset with comments and corresponding postive or negative sentiment. 
+            // When you create a loader you specify the schema by passing a class to the loader containing
+            // all the column names and their types. This will be used to create the model, and train it. 
+            pipeline.Add(new TextLoader<SentimentData>(TrainDataPath, useHeader: false, separator: "tab"));
+
+            // TextFeaturizer is a transform that will be used to featurize an input column. 
+            // This is used to format and clean the data.
+            pipeline.Add(new TextFeaturizer("Features", "SentimentText"));
```

Comment:
Would be a good opportunity to show off some of the hyperparameters of the TextFeaturizer.
```c#
pipeline.Add(new TextFeaturizer("Features", "SentimentText")
{
    WordFeatureExtractor = new NGramNgramExtractor() { NgramLength = 2, AllLengths = true },
    CharFeatureExtractor = new NGramNgramExtractor() { NgramLength = 3, AllLengths = false }      
});
```
Others of course are available: https://github.com/dotnet/machinelearning/blob/86f5ee663611f6c97ddb87be1150601018d74b4b/test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs#L54

Reply:
This makes sense in other samples we are adding or going to add. For getting started sample the easier the better.